examples/client/Locomotion/src/Components/AvailabilityVehicle/index.tsx

```diff
@@ -77,8 +74,14 @@ const AvailabilityVehicle = ({
       coordinate={locationAnimated}
       anchor={{ x: 0.5, y: 0.40 }}
     >
-      <SvgIcon Svg={carIcon} height={48} width={48} fill={primaryColor} />
+      <SvgIcon
+        Svg={carIcon}
+        height={48}
+        width={48}
+        style={{ transform: [{ rotate: `${location.bearing}deg` }], color: vehicleColor }}
```

Comment:
does it the same as the control center?

Reply:
what do you mean? cc doesn't have vehicle icons